.github/workflows/pr.yml

```diff
@@ -12,10 +12,10 @@ jobs:
         with:
           node-version: 18
           cache: "npm"
-      - name: Install dependencies
+      - name: Install dependencies & generate css types
         run: npm i --ci
       - name: Check types
-        run: npm run type-check
+        run: npm run type:check
```

Comment:
I think this is moot if they're not being checked in and generated within the same workflow?

Reply:
The css types are generated within the workflow, then the `type:check` (which is the regular typescript check) command will check for type errors throughout the repo. This way it'll effectively see if an imported css module is using an invalid property. 

The `type-css:check` command will still be available for us to use locally, in order to double check that our declaration files are updated.

Reply:
oh my bad, read too quickly